stacks/overrides-xhprof.yml

```diff
@@ -0,0 +1,34 @@
+# Adds Web IDE service to the project
+# The XHProf Outputs can be accessed at xhprof.${VIRTUAL_HOST}
+
+version: "2.1"
+
+services:
+
+  cli:
+    environment:
+      - XHPROF_ENABLED=1
+      - XHPROF_OUTPUT_DIR=${XHPROF_OUTPUT_DIR:-/tmp/xhprof}
+
+  # XHProf Service to review
+  xhprof:
+    image: ${XHPROF_IMAGE:-wodby/xhprof}
+    hostname: xhprof
+    expose:
```

Comment:
Expose is not necessary here, since the image already does that - https://github.com/wodby/xhprof/blob/master/Dockerfile#L36